api/package-lock.json

```diff
@@ -13,33 +13,18 @@
         "express": "^4.17.3",
         "jest": "^29.5.0",
         "jsonwebtoken": "^9.0.0",
-        "pg": "^8.9.0"
+        "pg": "^8.9.0",
+        "socket.io": "^4.6.2"
       },
       "devDependencies": {
         "nodemon": "^2.0.15",
         "supertest": "^6.3.3",
         "vitest": "^0.29.2"
       }
     },
-    "..": {
-      "name": "full-stack-react",
-      "version": "1.0.0",
-      "extraneous": true,
-      "dependencies": {
-        "bootstrap": "^5.2.3",
-        "express": "^4.18.2",
-        "jsonwebtoken": "^9.0.0",
-        "react-bootstrap": "^2.7.4"
-      },
-      "devDependencies": {
-        "concurrently": "^7.1.0",
-        "prettier": "^2.8.4"
-      }
-    },
     "node_modules/@ampproject/remapping": {
       "version": "2.2.1",
-      "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.1.tgz",
-      "integrity": "sha512-lFMjJTrFL3j7L9yBxwYfCq2k6qqwHyzuUl/XBnif78PWTJYyL/dfowQHWE3sp6U6ZzqWiiIZnpTMO96zhkjwtg==",
+      "license": "Apache-2.0",
```

Comment:
what is this license for?

Reply:
It's for a node package dependency - but it basically says no one can patent this and make money off of it https://en.wikipedia.org/wiki/Apache_License